devel/meson/Portfile

```diff
@@ -23,13 +23,13 @@ long_description    Meson is a build system designed to optimize programmer prod
                     Valgrind, CCache and the like. It is both extremely fast, and, even more importantly, \
                     as user friendly as possible.
 
-checksums           rmd160  36994a8e29e15751a9a1977aad581057a52f3943 \
-                    sha256  0a35cfd4cb779a6a415ad3ccbf2d4143334e96139125585ce49915bec494da02 \
-                    size    1953620
+checksums           rmd160  96db39f90328a64d508cc7c0075aa89fecde939b \
+                    sha256  a0f5caa1e70da12d5e63aa6a9504273759b891af36c8d87de381a4ed1380e845 \
+                    size    2034805
 
-# as of verison 0.45.0,requires python 3.5 or better
+# As of verison 0.62.0, requires python 3.7 or later
 
-python.versions         36 37 38 39
+python.versions         37 38 39
```

Comment:
Does this list actually serve any purpose given that there are no variants to pick from? Wouldn't it be the same if it contained just 310?

Reply:
So just change that to the following instead?

```
python.default_version 310
```

Reply:
I'm no longer sure how all these lists work (I should double-check by looking at the sources). This particular list is helpful when creating variants. When variants aren't created, I'm not sure if there's any advantage of listing these versions here (or at least I don't know how to make use of this list).

Reply:
I do not insist on keeping the list, but I have seen that others may get upset if something is changed without a clear necessity, even if a change appears for the good.

Reply:
> I do not insist on keeping the list, but I have seen that others may get upset if something is changed without a clear necessity, even if a change appears for the good.

I think the only sensible change for the moment would be adding python310 and setting this to default but not removing the list until there’s some confirmation.

I’ve been using the latest meson for a while now with python310 and not seen any issues with the Ports I’m using.

Reply:
> > I do not insist on keeping the list, but I have seen that others may get upset if something is changed without a clear necessity, even if a change appears for the good.
> 
> I think the only sensible change for the moment would be adding python310 and setting this to default but not removing the list until there’s some confirmation.
> 
> I’ve been using the latest meson for a while now with python310 and not seen any issues with the Ports I’m using.

Sounds good.

Reply:
the `python.versions` line doesn't do anything here (it only creates subports if the portname starts with `py-`; so keeping only the `python.default_version` line here seems correct

Reply:
@reneeotten @Gcenx @SoapZA I am indifferent towards keeping/removing `python.versions`, but please, I need a consistent opinion on whether to keep it or remove :)

Reply:
`python.version` is only used for `py-<name>` ports and doesn't do anything when the portname doesn't start with `py-` --> remove

Reply:
As meson doesn't have sub ports that make use of the other python versions it makes sense to just remove it.

Reply:
OK